truthy-falsy.js

```diff
@@ -0,0 +1,219 @@
+// 1
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(true, false));
+
+// 2
+function closeCompare(firstNumber, secondNumber, numbersMargin = 0){
+  const numbersDiff = Math.abs(firstNumber - secondNumber);
+  if (numbersDiff <= numbersMargin) {
+    return 0;
+  }
+  if (firstNumber < secondNumber) {
+    return -1;
+  }
+  return 1;
+}
+
+// 3
+function or(firstBoolean, secondBoolean) {
+  if (firstBoolean === true) {
+    return true;
+  }
+  if (secondBoolean === true) {
+    return true;
+  }
+  return false;
+}
+console.log(or(true, false));
+
+function xor(firstVariable, secondVariable) {
+  if (firstVariable && !secondVariable) {
+    return true;
+  }
+  if (secondVariable && !firstVariable) {
+    return true;
+  }
+  return false;
+}
+console.log(xor(true, true));
+
+// 4
+function isTruthy(booleanVariable, isBooleanTruth, isBooleanFalse) {
```

Comment:
The names `isBooleanTruth` and `isBooleanFalse` don't suggest that those arguments are functions.
Typically, a function passed into another function is called a **callback**